RELEASE_NOTES.md

```diff
@@ -15,3 +15,4 @@
 ## Bug Fixes
 
 * Change 0.0 to nan if there is no data available
+* Enforce keyword arguments in cumulative_energy function
```

Comment:
Nitpick, but this is not really a bug fix, I would say it's more of an improvement, and a breaking change, as now callers must use keyword arguments. So in the future you might want to mention it in the **Upgrading** section.